pypeit/sensfunc.py

```diff
@@ -128,7 +130,8 @@ class SensFunc(datamodel.DataContainer):
                  'steps',
                  'splice_multi_det',
                  'meta_spec',
-                 'std_dict'
+                 'std_dict',
+                 'spat_fwhm_std'
```

Comment:
We shouldn't allow an attribute to be both a component of the `datamodel` and an `internal`.  We should catch this when validating a `DataContainer`.